mcs/class/Mono.Messaging/Assembly/AssemblyInfo.cs

```diff
@@ -59,3 +59,4 @@
 
 [assembly: AssemblyFileVersion (Consts.FxFileVersion)]
 [assembly: CompilationRelaxations (CompilationRelaxations.NoStringInterning)]
+[assembly: InternalsVisibleTo ("Mono.Messaging_test_net_4_5, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
```

Comment:
Yeah, I knew it used to read from the environment. I think I prefer AppSettings, philosophically, since that's "the .NET way." Also, changing back to environment at this point would be a breaking change. Not to mention, it wouldn't solve my problem since I still need some way to reset the singleton instance.